ingestion/create_cadet_databases_source/source.py

```diff
@@ -279,11 +288,18 @@ def _get_databases_with_domains_and_display_tags(
                         logging.debug(f"{database} - has no database level metadata")
 
                     database_metadata_dict["domain"] = fqn[1]
+                    # Tags are a list which is unhashable for the tuple
+                    database_tags = database_metadata_dict.pop("tags", [])
                     database_metadata_tuple = tuple(database_metadata_dict.items())
                     database_mappings.add((database, database_metadata_tuple))
                     domain_lookup.set(database, table, database_metadata_dict["domain"])
 
                     tags = get_tags(manifest["nodes"][node])
+                    if database_tags:
+                        tags.extend(database_metadata_dict.get("tags", []))
```

Comment:
They've been popped so won't be there. Shouldn't this be
```suggestion
                    if database_tags:
                        tags.extend(database_tags)